frontend/src/components/search/ContactResultsTable.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import { Table, Row, Col, Divider } from "antd";
+import PropTypes from "prop-types";
+import Highlight from "react-highlighter";
+
+import { Link } from "react-router-dom";
+import * as router from "@/constants/routes";
+
+/**
+ * @class  ContactResultsTable - displays a table of mine search results
+ */
+
+const propTypes = {
+  header: PropTypes.string.isRequired,
+  highlightRegex: PropTypes.objectOf(PropTypes.regexp).isRequired,
+  searchResults: PropTypes.arrayOf(PropTypes.object).isRequired,
+  query: PropTypes.string.isRequired,
+  partyRelationshipTypeHash: PropTypes.objectOf(PropTypes.strings).isRequired,
+  showAdvancedLookup: PropTypes.bool.isRequired,
+};
+
+const defaultProps = {};
+
+const parseQuery = (query) => {
+  const emailExp = /@/;
+  const phoneExp = /^\d+(-\d*)*$/;
```

Comment:
We have email and phone RegEx's in `Validate.js`
Can we import those and use them in this component? ( I know we spoke in the past about making our email regex the barebones version, so the validate.js can be updated to use the above regEx for email)
- There are also existing checks in that file for phone and email.

Reply:
Sure, that makes sense!

Reply:
Changed!